CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+## v2.7.2
+
+*14 oct 2022*
+
+- Added support for PVR container (#423)
```

Comment:
this should be a added to `Added support for compressed texture`

Reply:
Think we should only link to the compressed texture PR #391 because we now have 2 almost identical lines in the changelog: 
```
Added support for compressed texture (#423)
...
Added texture compression support (#391)
```

Reply:
yeah agreed .. that's what I meant with my comment as well :)

We should only have 1 note about adding texture compression support. @sandeep-vedam let's remove line 7 of the change log